docs/source/essentials/schema.md

```diff
@@ -223,6 +223,10 @@ mutation HomepageQuickAddBook {
 }
 ```
 
+## Passing arguments as variables
+
+TODO
```

Comment:
There is also the opportunity to discuss this earlier, say in `Query`, but I thought this would be a great place to demonstrate it for both mutations and queries.